app/src/main/scala/uk/ac/wellcome/platformapi/controllers/ManagementController.scala

```diff
@@ -0,0 +1,27 @@
+package uk.ac.wellcome.platform.api.controllers
+
+import com.twitter.finagle.http.Request
+import com.twitter.finatra.http.Controller
+import uk.ac.wellcome.platform.api.services._
+
+import javax.inject.{Inject, Singleton}
+import com.sksamuel.elastic4s.ElasticDsl._
+import scala.concurrent.Future
+import org.elasticsearch.action.admin.cluster.health.ClusterHealthResponse
+import scala.concurrent.ExecutionContext.Implicits.global
+
+@Singleton
+class ManagementController @Inject()(
+  elasticsearchService: ElasticsearchService
+) extends Controller {
+
+  get("/management/healthcheck") { request: Request =>
+    response.ok.json(Map("message" -> "ok"))
+  }
+
+  get("/management/clusterhealth") { request: Request =>
+    elasticsearchService.client
+      .execute { clusterHealth() }
+      .map { response.ok.json }
+  }
```

Comment:
Open Q: do we want to expose these management endpoints? To end-users: definitely not. To internal staff: we should talk to James et al. about what they’re doing with healthcheck/monitoring and so on.